prog/weaver/main.go

```diff
@@ -687,7 +687,7 @@ func determineQuorum(initPeerCountFlag int, router *weave.NetworkRouter) uint {
 	// that resolve to the same peer, in which case the quorum
 	// might be larger than it needs to be.  But the user can
 	// specify it explicitly if that becomes a problem.
-	clusterSize := uint(len(peers) + 1)
+	clusterSize := uint(len(peers))
```

Comment:
Why did we lose the `+1` here?  The comment suggests it is the "conservative assumption"

Reply:
:man_facepalming: My bad. Unintended.

Actually wanted to remove in the call checkForUpdates. For this call len(peers) should be sufficient right?
``` go
checkForUpdates(dockerVersion, router, uint(len(peers)+1)
```

Reply:
Yeah, I would leave off the `+1` there.